buildchain/buildchain/salt_tree.py

```diff
@@ -259,6 +259,7 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/_modules/metalk8s.py'),
 
     Path('salt/_pillar/metalk8s.py'),
+    Path('salt/_pillar/metalk8s_certs.py'),
```

Comment:
Maybe rename the file and the variable and pillar key I think

Reply:
I'll go for `metalk8s_private` if that's fine with you

Reply:
Yes it's good